frontend/src/components/Utils/temp-auth-utils.js

```diff
@@ -1,58 +1,53 @@
 /**
  * @fileoverview This is a temporary file that is used to implement 'fake'
  * versions of the Auth utility functions used in the ViewTrips components.
+ * As of right now, the only 'real' auth function used is `getCurUserEmail()`
+ * but a fake user uid is still used due to the lack of a set of functions
+ * that can convert  user email <--> user uid.
  *
  * TODO(Issue #55): Remove this whole file function and replace any imports to
  *                 this file with Auth utils.
  */
-
-
-/**
- * Temporary hardcoded function that returns the current users email.
- *
- * @return {string} Hardcoded user email string.
- */
-export function getCurUserEmail() {
-  return 'matt.murdock';
-}
+import authUtils from '../AuthUtils';
 
 /**
- * Temporary hardcoded function that returns the current users uid.
+ * Temporary 'fake'/hardcoded function that returns the current users uid.
  *
- * @return {string} Hardcoded user uid string.
+ * @return {string} Fake user uid string the form '_userEmail_'.
  */
 export function getCurUserUid() {
-  return getUserUidFromUserEmail(getCurUserEmail());
+  return getUserUidArrFromUserEmailArr([authUtils.getCurUserEmail()]).pop();
```

Comment:
There is currently no way to just get the UID for 1 user, and an array must be created for that?

Reply:
There will be when it comes time to use the real Auth functions. But as of right now I had to make this fake function for consistency with the email <--> uid functions below. Although this implementation is VERY gross. So I just changed it to a more "hard-coded" way but is much easier to read.